www/css/main.diary.css

```diff
@@ -351,6 +371,10 @@ a.item-content {
   height: 50%;
 }
 
+.diary-infos:nth-child(2) {
+  justify-content: center;
+}
+
```

Comment:
`nth-child` is a css feature to style the nth sibling. In this case, it's the 
bottom half of the diary card which contains the button. To prevent text overlapping we needed to center it

The root problem of all this is that the UI relies on fixed pixel values for heights and sizes, and is not structured for responsive design. We cannot fix that without an overhaul, so this remedy tries to make things at least look presentable

Reply:
I did not know this! The styling aspects of HTML are the ones that I am least familiar with since I am not very good at visual design and have generally handed that off to others for the entire duration of the project.